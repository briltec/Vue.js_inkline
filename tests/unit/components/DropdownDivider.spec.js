import { mount } from '@vue/test-utils';
import DropdownDivider from 'inkline/components/DropdownDivider';

describe('Components', () => {
    describe('DropdownDivider', () => {
        let wrapper;

        beforeEach(() => {
            wrapper = mount(DropdownDivider);
        });

        test('should render correctly', () => {
            expect(wrapper.html()).toMatchSnapshot();
        });
    });
});
