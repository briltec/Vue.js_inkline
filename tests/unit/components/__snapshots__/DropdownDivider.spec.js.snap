// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components DropdownDivider should render correctly 1`] = `
<button tag="button" tabindex="1" class="button">
  <!---->
  <!---->
  <!----></button>
`;
